theme/mixins/focus.less

```diff
@@ -1,7 +1,7 @@
-.focus() {
+.focus(@focus-color: @sk-focus) {
```

Comment:
This parameter doesnt appear to be used, did it become unnecessary after a refactor?

Reply:
Yes it became unnecessary after a refactor. I decided to leave it in because it seems like something we will need at some point. But I'm happy to take it out - it's easy to add back in the future